### DISCUSSION TITLE
Can YARP be configured as a middleware in C# Blazor WebAssembly?

### DISCUSSION THREAD

**Discussion by jhunexjun** [dotnet/yarp#614] [Q&A] [answered]
My issue is I don't have an access to the web API, thus when I fetch data from Blazor WebAssembly I am getting CORS issue. I tried reading the docs but it doen't make sense a lot to me so I am asking help if anyone can provide a sample code if that's possible?

**Comment by Tratcher** [comment, marked as answer]
That's a question probably better answered by the Blazor team. Check out this thread: https://github.com/dotnet/aspnetcore/discussions/22988

**Reply by eriknyk** [reply]
the link is broken

**Reply by Tratcher** [reply]
We were able to capture the content of that thread from the archives:
[Discussion.pdf](https://github.com/microsoft/reverse-proxy/files/9914427/Discussion.pdf)
